packages/teams-js/src/public/meeting.ts

```diff
@@ -1,17 +1,9 @@
-import { sendAndHandleSdkError } from '../internal/communication';
+import { sendMessageToParent } from '../internal/communication';
 import { registerHandler } from '../internal/handlers';
 import { ensureInitialized } from '../internal/internalAPIs';
-import {
-  callCallbackWithErrorOrResultOrNullFromPromiseAndReturnPromise,
-  callCallbackWithSdkErrorFromPromiseAndReturnPromise,
-} from '../internal/utils';
 import { FrameContexts } from './constants';
 import { SdkError } from './interfaces';
-import { runtime } from './runtime';
 
-/**
- * @alpha
- */
 export namespace meeting {
```

Comment:
Similar to the [comments in the media.ts PR](https://github.com/OfficeDev/microsoft-teams-library-js/pull/1101#discussion_r855583974), can you please add @deprecated tags to the namespace and APIs referencing version 2.0.0-beta.5?